content/js/asset.md

```diff
@@ -1,55 +1,31 @@
-## Uploading an asset by base64 data
-
-```js
-// node.js
-fs.readFile('my-awesome-picture.jpg', (err, data) => {
-  if (err) {
-    throw err;
-  }
-
-  let profilePic = new skygear.Asset({data: data});
-  // Profile = skygear.Record.extend('profile');
-  let profile = new Profile({
-    name: 'Handsome Guy',
-    picture: profilePic
-  });
-  skygear.publicDB.save(profile).then((p) => {
-    // fetched record field will have a `url` attribute available
-    console.log('Download my handsome selfie here => %s', p.picture.url);
-  });
-})
-```
-
-## Creating an asset from `input` element
+For file upload to work properly, please set `ASSET_STORE_URL_PREFIX` environment
+variable to have value `https://<your-app-name>.skygeario.com/files` (which is
+just your Server EndPoint plus `files` at the end) on you Skygear
```

Comment:
- avoid using the word "just", you may use "simply" instead
- you -> yours
- the portal link should be applied to either "Settings page" or "Skygear Portal Settings page"